src/routes/PetsPage/PetsPage.js

```diff
@@ -1,34 +1,200 @@
 import React, { Component } from 'react';
+import Select from 'react-select';
 import PetCard from '../../components/PetCard/PetCard';
-import { Section } from '../../components/Utils/Utils';
+import { Input, Section } from '../../components/Utils/Utils';
 import PetsService from '../../services/petsService';
 
 export default class PetsPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      breeds: [],
       pets: [],
-      petCount: -1  // this is set to prevent the no pets alert before data is loaded
+      petCount: -1, // this is set to prevent the no pets alert before data is loaded
+      typeOfAnimal: [],
+      breed: [],
+      sex: '',
+      minAge: '',
+      maxAge: '',
+      more: []
     };
   }
 
   componentDidMount() {
-    PetsService.getPets()
+    PetsService.getBreeds()
```

Comment:
Do you think we can call `getBreeds()` in the `/shelters/shelterID/pets/create` page so that a user can choose from the existing breeds?

Reply:
Definitely, I actually forgot to change it